usr.sbin/adduser/adduser.conf.5

```diff
@@ -171,6 +171,14 @@ The default information to be held in the GECOS field of
 .It Va uidstart
 The default user ID setting.
 This must be a number above 1000 and fewer than 65534.
+.It Va Zflag
+Turn on ZFS support by default.
+.It Va zfsopt
+Options passed into the 
+.Xr zfs-create 8
```

Comment:
IMHO zfs create option should not be passed.  These datasets should inherit attributes from an upper level, especially compression, atime, etc.; encryption would be the only legitimate option to be passed.